include/oneapi/dpl/pstl/hetero/dpcpp/execution_sycl_defs.h

```diff
@@ -50,14 +52,34 @@ class device_policy
     device_policy(const device_policy<OtherName>& other) : q(other.queue())
     {
     }
+
+    device_policy(const device_policy& other) : q(other.queue()) {}
+    device_policy(device_policy&& other) : q(other.queue()) {}
+    device_policy&
+    operator=(const device_policy& other)
+    {
+        q = other.queue();
```

Comment:
Should we really need to create ```sycl::queue``` instance here in ```other.queue()``` call ?
May be good enough simple copy ```q``` state?

Reply:
It should be sufficient to just copy `q` directly as you suggest.

Reply:
Done.